corehq/apps/geospatial/tests/test_views.py

```diff
@@ -96,3 +134,103 @@ def test_config_update(self):
         )
         config = GeoConfig.objects.get(domain=self.domain)
         self.assertEqual(config.user_location_property_name, 'some_other_name')
+
+
+class TestGPSCaptureView(BaseGeospatialViewClass):
+
+    urlname = GPSCaptureView.urlname
+
+    def test_no_access(self):
+        response = self.client.get(self.endpoint)
+        self.assertEqual(response.status_code, 404)
+
+        # Logged in but FF not enabled
+        self.client.login(username=self.username, password=self.password)
+        response = self.client.get(self.endpoint)
+        self.assertEqual(response.status_code, 404)
+
+    @flag_enabled('GEOSPATIAL')
+    def test_success(self):
+        self.client.login(username=self.username, password=self.password)
+        response = self.client.get(self.endpoint)
+        self.assertEqual(response.status_code, 200)
+
+
+@flag_enabled('GEOSPATIAL')
+@es_test(requires=[case_search_adapter, user_adapter], setup_class=True)
+class TestGetPaginatedCasesOrUsers(BaseGeospatialViewClass):
+
+    urlname = 'get_paginated_cases_or_users_without_gps'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        case_type = 'foobar'
+        cls.case_a = create_case(
+            cls.domain,
+            case_id=uuid4().hex,
+            case_type=case_type,
+            name='CaseA',
+            save=True
+        )
+        cls.case_b = create_case(
+            cls.domain,
+            case_id=uuid4().hex,
+            case_type=case_type,
+            name='CaseB',
+            case_json={
+                GPS_POINT_CASE_PROPERTY: '12.34 45.67',
+            },
+            save=True,
+        )
+        case_search_adapter.bulk_index([cls.case_a, cls.case_b], refresh=True)
+
+        cls.user_a = CommCareUser.create(
+            cls.domain, 'UserA', '1234', None, None, metadata={GPS_POINT_CASE_PROPERTY: '12.34 45.67'}
```

Comment:
nit: good to use keyword arguments with long lists of args.
At times its not obvious what the argument is for, specially when using `None`

Reply:
[95cfd32](https://github.com/dimagi/commcare-hq/pull/33420/commits/95cfd327ad38124044621b5283032b55fb91028f)